infrastructure/templates/public-api/main.bicep

```diff
@@ -0,0 +1,372 @@
+param seedRegistry bool = true
+
+//ServiceBus Queue Params -------------------------------------------------------------------
+@description('Name of the Service Bus namespace')
+param namespaceName string = 'etlnamespace'
+
+@description('Name of the Queue')
+param queueName string = 'etlfunctionqueue'
+
+//ETL Function Paramenters ------------------------------------------------------------------
+@description('Application : Insights name')
+param applicationInsightsName string = 'etlFunctionInsights'
+
+@description('Function App Plan operating system')
+@allowed([
+  'Windows'
+  'Linux'
+])
+param appServiceplanOS string = 'Linux'
+
+@description('Function App : Runtime Language')
+@allowed([
+  'dotnet'
+  'node'
+  'python'
+  'java'
+])
+param functionAppRuntime string = 'python'
+
+
+//---------------------------------------------------------------------------------------------------------------
+// All resources via modules
+//---------------------------------------------------------------------------------------------------------------
+
+//Deploy Networking
+module vnetModule 'components/network.bicep' = {
+  name: 'virtualNetworkDeploy'
+  params: {
+    subscription: subscription
+    location: location
+    environment: environment
+    deploySubnets: deploySubnets
+    //tags
+    departmentName: departmentName
+    environmentName: environmentName
+    solutionName: solutionName
+    subscriptionName: subscriptionName
+    costCentre: costCentre
+    serviceOwnerName: serviceOwnerName
+    dateProvisioned: dateProvisioned
+    createdBy: createdBy
+    deploymentRepo: deploymentRepo
+    deploymentScript: 'network.bicep'
+  }
+}
+
+//Deploy Storage Account
+module storageAccountModule 'components/storageAccount.bicep' = {
+  name: 'storageAccountDeploy'
+  params: {
+    subscription: subscription
+    location: location
+    adminSubnetRef: vnetModule.outputs.adminSubnetRef
+    importerSubnetRef: vnetModule.outputs.importerSubnetRef
+    publisherSubnetRef: vnetModule.outputs.publisherSubnetRef
+    storageFirewallRules: storageFirewallRules
+    //tags
+    departmentName: departmentName
+    environmentName: environmentName
+    solutionName: solutionName
+    subscriptionName: subscriptionName
+    costCentre: costCentre
+    serviceOwnerName: serviceOwnerName
+    dateProvisioned: dateProvisioned
+    createdBy: createdBy
+    deploymentRepo: deploymentRepo
+    deploymentScript: 'storageaccount.bicep'
+  }
+  dependsOn: [
+    vnetModule
+  ]
+}
+
+//Deploy File Share
+module fileShareModule 'components/fileShares.bicep' = {
+  name: 'fileShareDeploy'
+  params: {
+    fileShareName: fileShareName
+    fileShareQuota: fileShareQuota
+    storageAccountName: storageAccountModule.outputs.storageAccountName
+    //tags
+  }
+  dependsOn: [
+    storageAccountModule
+  ]
+}
+
+//Deploy Function blob store
+module blobStoreModule 'components/fileShares.bicep' = {
```

Comment:
should this be `components/blobStore.bicep`?

Reply:
Yes correct, it should be